### DISCUSSION TITLE
Missed Reading Alerts Not Audible

### DISCUSSION THREAD

**Discussion by Lynd** [Navid200/xDrip#63] [Dexcom]
Recently, when I have "Do not Disturb"  set on my phone, Missed Reading alerts are not audible.    I have "Override Silent Mode" selected in Other Alerts and I'm using Android 9.   @Navid200, you had suggested that I test by "disabling collection" .  Can you tell me how to do this?

**Comment by Navid200** [comment]
To test the missed reading alert on a follower you can just disable the follower details.  For example, you can intentionally change the password entered in the follower temporarily to something wrong, or you can just disable the follower scheme altogether.

To test the missed reading alert on a collector (master), you can tap on:
Settings -> Hardware Data Source -> Disable Collection

Of course this will stop your readings.  So, I suggest you reduce the time threshold of the missed reading alert so that it would trigger right away so that you can be done with the test soon.   So, on the missed readings alert settings page, you can change the "Alert if no data received in:" setting to something like 7 minutes.  

It helps to know if the alert triggers or not.  You can find that out looking at the logs.
If the alert isn't triggering or if it does but you are not hearing a sound, the problem isn't going to be the same.

**Reply by Lynd** [reply]
Thank you.  I followed these instructions and got a few alerts triggered, but no audio.

**Reply by Navid200** [reply]
Would you please show the event log that shows the alert trigger that you did not hear?

**Reply by Lynd** [reply]
Sure. I'm not sure how to do that though.  Do you want me to select "upload event logs" ?

**Reply by Navid200** [reply]
No,

This is where the logs are:
https://github.com/Navid200/xDrip/wiki/Logs

Please take a snapshot.  Different phones have different ways.  You can find out the easiest way to create a snapshot of what is shown on screen using your phone.

Save the snapshot somewhere you can find.  
When you are editing your post, you can attach the image to your post.

Let me know if anything is not clear.

**Reply by Navid200** [reply]
You can scroll up and down when you are in the xDrip logs to see the logs at different times.  
So, it's enough to scroll.  Find the log, and take one screenshot.

**Reply by Lynd** [reply]
![Screenshot_20210812-123224 (1)](https://user-images.githubusercontent.com/8272764/129244153-bf037b0f-b767-4565-9086-0d006f5b6382.png)

**Comment by Navid200** [comment]
Would you please go to Android settings.  This is not in xDrip.  This is just your phone.
Can you go to Settings -> Apps & notifications -> Notifications 
You should see a list of apps at the bottom of the screen.  Tap on xDrip in the last.

Can you show what you see?

**Reply by Lynd** [reply]
That seems to identify the problem 
![signal-2021-08-12-125630](https://user-images.githubusercontent.com/8272764/129246163-cbffd8df-2ac8-4c1e-8f24-719d544e3d91.png)

**Reply by Navid200** [reply]
Let's enable Override do not disturb  on that page and then repeat the test please.

**Reply by Lynd** [reply]
Now, the alerts include audio. Seems like it worked!

**Reply by Navid200** [reply]
Awesome
